traveler-swift-core/TravelerKit/TravelerRoute.swift

```diff
@@ -71,28 +71,24 @@ extension PassengerRoute: Route {
         }
 
         guard let errorJSON = try? JSONSerialization.jsonObject(with: data, options: .allowFragments) as? [String: Any],
-           let errorCode = errorJSON["errorCode"] as? Int else {
+            let errorCode = errorJSON["errorCode"] as? Int else {
             Log("Bad JSON", data: String(data: data, encoding: .utf8), level: .error)
             return error
         }
 
-        switch (errorCode, errorJSON["errorData"]) {
-        case (2006, _):
+        switch errorCode {
+        case 2006:
             return BookingError.noPasses
-        case (2007, _):
+        case 2007:
             return BookingError.veryOldTraveler
-        case (2012-2014, _):
+        case 2012-2014:
```

Comment:
Syntax should be `...` not `-` unless you have a custom operator defined somewhere?

Reply:
Twas a brainfart.